src/main/resources/jenkins/plugins/git/GitStep/help.html

```diff
@@ -0,0 +1,11 @@
+<div>
```

Comment:
AFAIR jenkins wraps all helps into divs and you don't need wrote it in help files.